src/components/map/MapService.js

```diff
@@ -765,7 +765,8 @@
                     layer.minResolution),
                 tileLoadFunction: tileLoadFunction,
                 url: getWmtsGetTileUrl(layer.serverLayerName,
-                  layer.format)
+                  layer.format),
+                crossOrigin: 'anonymous'
```

Comment:
The hit detection creates a canvas then try to export the data, and because of CORS you must set the crossOrigin to anonymous to allow the export.